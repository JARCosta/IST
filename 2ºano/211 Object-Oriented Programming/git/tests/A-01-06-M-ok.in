1
a.dat
0
