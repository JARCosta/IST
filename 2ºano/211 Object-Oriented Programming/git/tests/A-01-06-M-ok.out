Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: Ficheiro a abrir: Abrir: Operação inválida: Problema ao abrir 'a.dat'.
Menu Principal
1 - Abrir
2 - Guardar
3 - Mostrar Data
4 - Avançar Data
5 - Gestão de Produtos
6 - Gestão de Parceiros
7 - Gestão de Transacções
8 - Menu de Consultas
9 - Ver Saldo Global
0 - Sair
Escolha uma opção: 